Blackjack/GameResult.cs

```diff
@@ -20,7 +20,7 @@ public GameResult(IHand dealerHand, IHand playerHand)
 
         public int DealerScore { get => Score.Calculate(_dealerHand); }
         public int PlayerScore { get => Score.Calculate(_playerHand); }
-        public Outcome Outcome
+        public Outcome Outcome // TODO: look at changing this to a method

```

Comment:
There's some interesting discussions in this thread about when to use properties vs methods: https://stackoverflow.com/questions/601621/properties-vs-methods

My argument for changing this to a method is that there's logic & computation involved, rather than just retrieving a value. @Kedoodle would like to hear your rationale behind using a getter vs a method for this

Reply:
I have no particular preference, as long as the getter doesn't change any values i.e. maintains CQS.